src/lxc/cmd/lxc_user_nic.c

```diff
@@ -1039,7 +1041,7 @@ static bool is_privileged_over_netns(int netns_fd)
 	pid_t pid_self;
 	uid_t euid, ruid, suid;
 	bool bret = false;
-	int ofd = -1;
+	int ofd;
```

Comment:
Same here, uninitialized int type should go first in the declaration block here.

Reply:
Thanks!